### PR TITLE
Add service.yml for adding new project to semaphore CI

### PR DESCRIPTION
**Pull request by jliunyu**
For semaphore CI migration, we need to first add new project to semaphore  CI.

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+lang_version: 1.18
+# Add the following 2 lines if you want the automatically generated .semaphore files
+git:
+  enable: true
+github:
+  enable: true
+  repo_name: confluentinc/confluent-kafka-go # for example, confluentinc/cc-new-service
+semaphore:
+  enable: true
+
```
Comment (xli1996) [inline service.yml:12]: nit: extra line.

**Review by xli1996 — APPROVED** [review]

File: service.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: confluent-kafka-go
+lang: go
+lang_version: 1.18
+# Add the following 2 lines if you want the automatically generated .semaphore files
```
Comment (edenhill) [inline service.yml:4]: remove

File: service.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+git:
+  enable: true
+github:
+  enable: true
+  repo_name: confluentinc/confluent-kafka-go # for example, confluentinc/cc-new-service
```
Comment (edenhill) [inline service.yml:9]: remove comment

**Review by edenhill — CHANGES_REQUESTED** [review]

**Review by edenhill — APPROVED** [review]
LGTM

**Pull request merged**